.github/workflows/test.yml

```diff
@@ -65,10 +65,19 @@ jobs:
       run: cargo build --verbose
     - name: Run tests
       run: cargo test --verbose --workspace
+    - run: apt-get update && apt-get install jq -y
+    - name: build integration-test binary for subxt
+      run : eval "cp $(cargo test --no-run --message-format=json | jq '. | select(.executable != null) | .executable') test-binary"
+      working-directory: ./integration/subxt-tests
     - uses: actions/upload-artifact@master
       with:
         name: solang-linux-x86-64
         path: ./target/debug/solang
+    - uses: actions/upload-artifact@master
+      with:
+        name: test-binary
```

Comment:
The name is not very descriptive, `substrate-subxt-tests`  or so would be much better. Even better would be not to copy this file around